# This configuration works with file: 

right_arm_joints:
  - joint84     # Hand
  - joint77     # Elbow
  - joint70     # Shoulder
  - joint63     # Inner Shoulder

left_arm_joints:
  - joint56     # Hand 
  - joint49     # Elbow
  - joint42     # Shoulder
  - joint35     # Inner Shoulder

spine:
  - joint70     # Pelvis Low
  - joint7      # Pelvis High
  - joint14     # Back
  - joint21     # Neck

head:
  - joint28     # Head

labels:
  
  rWrist: joint84
  rElbow: joint77
  rShoulder: joint70
  lWrist: joint56

  lElbow: joint49
  lShoulder: joint42
  Pelvis: joint70
